### DISCUSSION TITLE
Help setting up vite-plugin-ssr on existing repo using Solid Start and Electron

### DISCUSSION THREAD

**Discussion by nikivdev** [vikejs/vike#1011] [Discussion]
I would love to use vite-plugin-ssr on my project, code here:

https://github.com/learn-anything/learn-anything

The issue I have and reason for wanting vite-plugin-ssr is that I currently have 2 packages, one with [Solid Start](https://github.com/solidjs/solid-start) code [here](https://github.com/learn-anything/learn-anything/tree/main/app/packages/website).  

But I also have a separate solid website [here](https://github.com/learn-anything/learn-anything/tree/main/app/packages/renderer) that does not use Solid Start but instead exposes `index.html` file and is built from default vite solid template. Electron app renderer process needs `index.html` which Solid Start can't provide (entry-client.tsx and entry-server.tsx are used instead).

Can vite-plugin-ssr help solve this problem of having one solid setup like in solid start that exposes `index.html` file? 

Thank you.

**Comment by nikivdev** [comment]
I am trying to make it work but there was [no solid example in docs](https://vite-plugin-ssr.com/add).

Will try replicate [Inlang](https://github.com/inlang/inlang) setup.

**Reply by brillout** [reply]
See https://vite-plugin-ssr.com/solid and https://github.com/magne4000/vike-solid/tree/main/vike-solid/renderer.

**Comment by brillout** [comment]
> Electron app renderer process needs `index.html`

If you pre-render your pages then you'll get `.html` files at `dist/client/`.

Keep us updated how your Electron integration is going and let me know if you believe there is a blocker on vite-pugin-ssr's side.

**Comment by rtritto** [comment]
@nikitavoloboev any update to use Vike (previously vite-plugin-ssr) + Solid + Electron or if there is a framework to use/help?

With Next.js (React) + Electron there is [nextron](https://github.com/saltyshiomix/nextron).
